test/e2e/suites/e2e/e2e_test.go

```diff
@@ -891,23 +890,43 @@ func isCloudProviderInitialized(taints []corev1.Taint) bool {
 	return true
 }
 
-func createTestVolumeType(e2eCtx *shared.E2EContext) {
+func findVolumeTypeID(volumeClient *gophercloud.ServiceClient, volumeTypeName string) (string, error) {
+	allPages, err := volumetypes.List(volumeClient, volumetypes.ListOpts{}).AllPages()
+	if err != nil {
+		return "", err
+	}
+
+	allVolumeTypes, err := volumetypes.ExtractVolumeTypes(allPages)
+	if err != nil {
+		return "", err
+	}
+
+	for _, volumeType := range allVolumeTypes {
+		if volumeType.Name == volumeTypeName {
+			return volumeType.ID, nil
+		}
+	}
+
+	return "", fmt.Errorf("volume type '%s' not found", volumeTypeName)
+}
+
+func getVolumeType(e2eCtx *shared.E2EContext) {
 	providerClient, clientOpts, _, err := shared.GetAdminProviderClient(e2eCtx)
```

Comment:
Do we need admin privileges? I was hoping we can get rid of them completely so we can run the tests also when we do not have admin access

Reply:
No it seems to work fine when using the function `GetTenantProviderClient` instead. I got lost in the code there and forgot about this aspect of the issue. Thanks for reminding me.

Reply:
No need for neither.